cfgov/jinja2/v1/_includes/related-content.html

```diff
@@ -24,7 +24,7 @@
 <div class="related-links">
     <h2 class="header-slug">
         <span class="header-slug_inner">
-            Related links
+            Related content
```

Comment:
I'm not entirely sure how this file is used, but everything else in it is pretty link-specific, so I think I would revert this change.